ci/scripts/shipit

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -eu
+
+header() {
+	echo
+	echo "###############################################"
+	echo
+	echo "$*"
+	echo
+}
+
+: "${REPO_ROOT:?required}" # Contains the Git repo
+: "${RELEASE_NAME:?required}" # Contains release name
+: "${RELEASE_ROOT:?required}" # Contains any information that is passed on to subsequent steps, e.g. GitHub publish
+: "${GCP_SERVICE_KEY:?required}" # The GCP service key for accessing the blobstore, written to a temporary private.yml.
+: "${GITHUB_TOKEN:?required}" # The github access token
+: "${DRY_RUN:=false}" # Whether to do a dry run or a real release
+
+###############################################################
+
+CONCOURSE_ROOT="$(pwd)"
+export CONCOURSE_ROOT
+
+DRY_RUN_ARG="--dry-run=$DRY_RUN"
```

Comment:
you _could_ init that to a default, e.g. always do dry run if not explicitly disabled, or the other way around.

e.g.
```suggestion
DRY_RUN=${DRY_RUN:-false}
DRY_RUN_ARG="--dry-run=$DRY_RUN"
```

Reply:
we do that. see line 18